src/pages/Scholarships.jsx

```diff
@@ -13,6 +13,26 @@ import ScholarshipList from '../components/ScholarshipList';
 import FilterBar from '../components/FilterBar';
 import qParams from '../lib/QueryParams';
 
+const sortOptions = {
+  deadlineAsc: {
```

Comment:
I forgot to mention, I think you can keys like `deadline.asc` if it's in quotes, e.g. `'deadline.asc'`. I imagine you chose this route because it wasn't letting you with the dot (since there is no variable called `deadline`) but if you surround it with quotes it should work.

Reply:
Oh ok yea that was way I used the uppercase letter on the `dir`, but thats good to know. I did go the route of using the dot to separate the `field` and `dir` using the way you mentioned.